containers/ws/default-bastion.conf

```diff
@@ -1,6 +1,6 @@
 [WebService]
 ProtocolHeader = X-Forwarded-Proto
-LoginTitle = Cockpit Bastion
+LoginTitle = Cockpit
```

Comment:
Okay -- but please split this out into a separate commit "cockpit/ws: Change bastion login title", as it's independent from the README updates.